Makefile

```diff
@@ -382,6 +415,12 @@ yq: $(YQ) ## Download yq locally if necessary.
 $(YQ): | $(LOCALBIN)
 	$(call go-install-tool,$(YQ),github.com/mikefarah/yq/v4,${YQ_VERSION})
 
+.PHONY: cloud-nuke
```

Comment:
Not necessarily, we could write our own but it wouldn't be trivial.  I'd like to just add this to our script, but this will do for now; it was specifically requested in [#222 ](https://github.com/Mirantis/hmc/issues/212#:~:text=Post%2Dcleanup%20with%20aws%2Dnuke%20or%20a%20similar%20tool)

Reply:
I meant here that I didn't see any use of cloud-nuke (maybe I overlooked it). I do see that you have a script and some aws cli stuff, so probably there is no need to install it.